ai4pkm_cli/cli.py

```diff
@@ -390,7 +390,9 @@ def show_default_info(self):
                     agent_display = f" [green]\\[{agent_shortcuts.get(agent_name, agent_name)}][/green]"
                 else:
                     agent_display = f" [dim]\\[default][/dim]"
-                self.console.print(f"  {i}. {job.get('inline_prompt', 'Unknown')} - {job.get('cron', 'No schedule')}{agent_display}")
+                # Get job name from either inline_prompt or command
+                job_name = job.get('inline_prompt') or job.get('command', 'Unknown')
```

Comment:
`title = f'"{inline_prompt}"' if inline_prompt else f"\\[{command}]”`

This convention was used for other places. In later PR, we probably want to make a better abstraction for command and prompt.